demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -13,6 +13,7 @@
 from git import GitCommandError, Repo
 from packaging.version import parse
 
+import demisto_sdk.commands.common.tools_paths as tools_paths
```

Comment:
why the `as`? I think it'd work without it

Reply:
[Indeed](https://github.com/demisto/demisto-sdk/pull/3324/commits/13672f497c4bbc37e2efa6c5c150f447c2c3ad72).